server/controllers/templates/web_templates.go

```diff
@@ -441,7 +441,7 @@ var ProjectJobsTemplate = template.Must(template.New("blank.html.tmpl").Parse(`
     <title>atlantis</title>
     <meta name="description" content>
     <meta name="author" content>
-    <link rel="stylesheet" href="{{ .CleanedBasePath }}/static/css/xterm.css">
+    <link rel="stylesheet" href="{{ .CleanedBasePath }}/static/css/xterm-5.3.0.css">
```

Comment:
Could we add renovate comment to these portions of the code so they get auto updated by the bot?

Reply:
Don't know how. Those files need to be manually extracted from the xterm.js npm package.

Reply:
It would be a package manager (file parser) and a data source (npm) to get the latest version. Thee would be defined in .github/renovate.json5

https://github.com/runatlantis/atlantis/blob/main/.github/renovate.json5

---

We can always do it in a follow up pr

Reply:
I created this issue for now for future ref

https://github.com/runatlantis/atlantis/issues/4286